examples/run_CPMG.py

```diff
@@ -31,91 +31,71 @@
 # {{{set phase cycling
 phase_cycling = True
 if phase_cycling:
-    ph1_cyc = r_[0, 1, 2, 3]
-    nPhaseSteps = 4
+    Nph_overall = r_[0,1,2,3]
+    Nph_diff = r_[0,2]
+    ph1_cyc = array([(j+k)%4 for k in Nph_overall for j in Nph_diff])
+    ph2_cyc = array([(k+1)%4 for k in Nph_overall for j in Nph_diff])
+    nPhaseSteps = len(Nph_overall) * len(Nph_diff)
 if not phase_cycling:
     ph1_cyc = 0.0
+    ph2_cyc = 0.0
+    Nph_overall = r_[0]
+    Nph_diff = r_[0]
     nPhaseSteps = 1
 # }}}
-# {{{better tau
-marker = 1.0  # 10/10/22 → what is this? → pretty sure the time needed to execute the marker command
-pad_start = config_dict["tau_extra_us"] - config_dict["deadtime_us"]
-pad_end = (
-    config_dict["tau_extra_us"] - config_dict["deblank_us"] - marker
-)  # marker + deblank
-assert(pad_start > 0), "tau_extra_us must be set to more than deadtime and more than deblank!"
-assert (pad_end > 0), "tau_extra_us must be set to more than deadtime and more than deblank!"
-twice_tau_echo_us = (  # the period between 180 pulses
-    config_dict["tau_extra_us"] * 2 + config_dict["acq_time_ms"] * 1e3
-)
-# now twice_tau_echo_us/2.0 is τ_echo, so I need to subtract the extra delays
-# imposed by the ppg to determine the remaining τ that I need
-config_dict["tau_us"] = twice_tau_echo_us / 2.0 - (
-    2
-    * config_dict["p90_us"]
-    / pi  # evolution during pulse -- see eq 6 of coherence paper
-    + config_dict["deadtime_us"]  # following 90
-    + config_dict["deblank_us"]  # before 180
-)
+# {{{symmetric tau
+marker = 1.0
+jumpto = 1.0
```

Comment:
I think call this `short_delay_us`, and use for both.  This solves two problems:

- do we really need two variables, since we want them both as short as possible?
- these are not named in a way that makes it clear what their units are.